client/CMT.cpp

```diff
@@ -1330,50 +1337,26 @@ static void mainLoop()
 	}
 }
 
-void startGame(StartInfo * options, CConnection *serv)
+void startGame()
 {
-	if(!settings["session"]["donotstartserver"].Bool())
-	{
-		serverAlive.waitWhileTrue();
-		serverAlive.setn(true);
-	}
-
-	if(settings["session"]["onlyai"].Bool())
-	{
-		auto ais = vm.count("ai") ? vm["ai"].as<std::vector<std::string>>() : std::vector<std::string>();
-
-		int i = 0;
-
-
-		for(auto & elem : options->playerInfos)
-		{
-			elem.second.playerID = PlayerSettings::PLAYER_AI;
-			if(i < ais.size())
-				elem.second.name = ais[i++];
-		}
-	}
-
-    client = new CClient();
+	client = new CClient();
+	CSH->client = client;

```

Comment:
1) startGame() should be moved to CServerHandler
2) no need to have static `client` pointer. Isn`t CSH->client enough?

Reply:
This one will still require some work since there is CMT with debug console and I don't know yet how to improve it.